HACKING.adoc

```diff
@@ -272,25 +272,34 @@ PR.
 You do not need to be an INRIA employee to open an account on this
 jenkins service; anyone can create an account there to access build
 logs, enable email notifications, and manually restart builds. If you
-would like to do this but have trouble doing it, please contact Damien
-Doligez or Gabriel Scherer.
+would like to do this but have trouble doing it, please contact
```

Comment:
Micro-nit - given that it's now an email address rather than a list of names, s/contact/email